src/Nox.Types/Types/Guid/Guid.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections.Generic;
+
+namespace Nox.Types;
+
+/// <summary>
+/// Represents a Nox <see cref="Guid"/> type and value object.
+/// </summary>
+public sealed class Guid : ValueObject<System.Guid, Guid>
+{
+    /// <summary>
+    /// A instance of the <see cref="Guid"/> object whose Value is all zeros.
+    /// </summary>
```

Comment:
Create it from Sytem.Guid.Empty no need for string hardcoded

Reply:
Updated to create it from `System.Guid.Empty`.